client/striplighttimer.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import time
+import datetime
```

Comment:
```suggestion
import datetime
import time

import requests
```
Python style is built-in first, alphabetically, a blank line, then other imports alpha sorted.  Note that VSCode can do this for you, you probably just need to install/enable style checking.